Dockerfile

```diff
@@ -8,11 +8,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+ARG BUILDER=golang:1.15-alpine
```

Comment:
This version of Go is fine for this PR, but we really should bump this in the `go.mod` and here

Reply:
makes sense. Lets do that separately to keep concerns separate. wdyt?

Reply:
we'll make a fast follow PR for updating go version

Reply:
+1, will raise another PR for the update.
Thank you so much.